chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts

```diff
@@ -11,7 +11,7 @@ export class FormValidationService {
     return true;
   }
 
-  private hasData(input: string): boolean {
+  public hasData(input: string): boolean {
```

Comment:
What did you change this method accessor for private to public?

Reply:
I had changed it on accident, but I reverted it back to private.